game-core/src/main/java/games/strategy/triplea/delegate/SpecialMoveDelegate.java

```diff
@@ -177,9 +177,6 @@ static MoveValidationResult validateMove(final Collection<Unit> units, final Rou
         result.addDisallowedUnit("Can only move owned units", unit);
       }
     }
-    if (validateAirborneMovements(data, units, route, player, result).getError() != null) {
-      return result;
-    }
```

Comment:
This might cause problems if validateAirborneMovements modifies the result object.

Reply:
:+1: for the observation. :-1: for side effects. :disappointed: